administrator-documentation/moderne-dx/how-to-guides/dx-configuration.md

```diff
@@ -10,53 +10,111 @@ Please talk to your sales representative or [contact us](mailto:support@moderne.
 
 ### Step 2: Determine how you will run the service
 
-Moderne offers two ways of running the service:
+There are two ways you can run the DX service:
 
-1. A Spring Boot executable JAR that can be run with Java
-2. (Coming soon) An [OCI image](https://github.com/opencontainers/image-spec) that can be run using any OCI runtime (e.g., Docker, Podman)
+1. You can use Java to run a Spring Boot executable JAR that we will provide you with
```

Comment:
DX is available on maven centrol https://central.sonatype.com/artifact/io.moderne/moderne-dx/versions. We can probably provide this link and say they can find the latest jar in maven central?

Reply:
Thanks for the suggestion! Updated to include that.